packages/v4/src/content/design-guidelines/components/tabs/tabs.md

```diff
@@ -2,4 +2,64 @@
 id: Tabs
 section: components
 ---
-A **tab** component creates a set of tabs to organize content on a page. It must always be used together with a tab content component.
+**Tabs** allow users to navigate between views within the same page or context.
+
+## Usage
+
+PatternFly offers a few types of tabs:
+
+- [Default tabs](#default-tabs) (recommended)
+- [Boxed tabs](boxed-tabs)
+- [Vertical tabs](#vertical-tabs)
+
+You can use as many tabs as you want in a page, although it is recommended you keep the total number of tabs to a minimum for a better user experience.If you have more tabs than fit on the page, you may use a [tab overflow](/components/tabs#default-overflow) which would allow a user to horizontal scroll through your tabs. The uses of both default and boxed stylings are flexible, so use either according to your needs and preferences. 
+
+### Default tabs
+
+Default tabs do not have any borders, other than the bottom line to distinguish between a selected tab, a hovered over tab, and a non-active tab. These tabs are most commonly used as top page header tabs or nested within components.
+
+<img src="./img/default-tabs.png" alt="Default tabs ex" width="1500"/>
+
+Default tabs can be formatted as standard or filled.  Filled tabs are placed side by side and span the whole width of the page. They can be used in screens with narrower widths, such as a drawer within a [primary detail view](/demos/primary-detail). When in doubt, stick with the standard styled tabs. 
+
+<img src="./img/default-vs-filled-tabs.png" alt="Standard and filled tabs ex" width="1500"/>
+
+1. Standard styled tabs
+2. Filled styled tabs
+
+### Boxed tabs
+
+Box tabs are functionally the same as default tabs, but they are outlined by a box to emphasize the area that the tab spans. Boxed tabs can be used interchangeably with default tabs, but we recommend using the default tabs over boxed tabs. Boxed tabs can be used in combination with default tabs to create more hierarchy on the page.
+
+<img src="./img/boxed-tabs.png" alt="Boxed tabs ex" width="1500"/>
+
+### Vertical tabs
+
+Vertical tabs are placed on the left hand side of a page or container. 
+
+<img src="./img/vertical-tabs.png" alt="Vertical tabs ex" width="1500"/>
+
+## Hierarchy
+
+Both boxed and default tabs can help create hierarchy on the page. We recommend using default tabs at all times, although mixing the two styles can separate sections of the page more distinctly. When using many levels of default tabs, make sure to incorporate the following types.
```

Comment:
Re-reading this, I feel like it's a little bit confusing because there is no clear distinction between what a primary tab is as opposed to a boxed tab or default tab?

I maybe would re-word to say, "When you have more than one tab level in your UI, you can either chose to use a mixture of boxed and default tabs to create hierarchy on a page, or you can chose to use primary and secondary tabs which differ in their font size, with primary tabs using 16pt font, and secondary tabs using 14pt font."

Or something like that? It's definitely hard to put into words/explain though, but do you see what I mean?

Maybe i'm misunderstanding the whole point of this section though. Are you just talking about information hierarchy in general, and not necessarily a "2 tab level" situation, or are you only talking about 2 tab level situations? or both?

Reply:
Added some clarification to why you'd use one over the other - let me know what you think @mmenestr

Reply:
Looks good, thank you!!